### PR TITLE
Wishlist

### PR DESCRIPTION
**Pull request by mohammadkhizerkhan**
added wishlist management

### CODE REVIEW

File: src/components/cart/Cart.jsx [hunk 1]
```diff
@@ -65,33 +72,80 @@ function Cart() {
                       <header class="card-header">
                         <h3>{title}</h3>
                         <h4>
-                        &#8377; {price} &nbsp; <span class="text-strike"> &#8377;({original_price})</span> &nbsp; <span>({Math.trunc(calculateDiscount(price,original_price))}%)</span>
+                          &#8377; {price} &nbsp;{" "}
```
Comment (Pratik1005) [inline src/components/cart/Cart.jsx:75]: Just as suggestion: Avoid using nbsp and {" "}, instead use elements for new line and css for spacing.

File: src/components/productpage/productlisting/Product.jsx [hunk 2]
```diff
@@ -21,36 +27,42 @@ function Product({item}) {
...
+
+  // const removeCartHandler = () => {
+  //   if (token) {
+  //     // navigate("/carts")
+  //     removeFromCart(token, item, cartDispatch);
+  //   } else {
+  //     navigate("/login");
+  //   }
+  //   console.log("removed");
+  // };
```
Comment (Pratik1005) [inline src/components/productpage/productlisting/Product.jsx:47]: You can remove commented code if you don't  need.

File: src/components/wishlist/Wishlist.jsx [hunk 3]
```diff
@@ -1,139 +1,99 @@
...
+  const { token } = useAuth();
+  const navigate = useNavigate();
+
+  const moveToCartHandler = (wishlistItem) => {
+    // cartState.cart.some((cartItem)=>cartItem._id===wishlistItem._id)?(
+    //   updateCartHandler(token, wishlistItem, ACTION_TYPE.INCR_QTY, cartDispatch)
+    // ):(
+    //   // console.log("hi from add")
+    //   addToCart(token, wishlistItem, cartDispatch)
+    //   );
```
Comment (Pratik1005) [inline src/components/wishlist/Wishlist.jsx:22]: same here

**Review by Pratik1005 — APPROVED** [review]
Other than suggested changes, good to go

**Review by Nikhil-Nirmal — APPROVED** [review]
Remove Unnecessary comments.

**Comment by Nikhil-Nirmal** [comment]
Everything else looks good

**Pull request merged**